examples/reshare/src/validator.rs

```diff
@@ -393,8 +393,11 @@ mod test {
             success_rate: 1.0,
         };
         for seed in 0..5 {
-            let state = all_online(5, seed, link.clone(), 25);
-            assert_eq!(state, all_online(5, seed, link.clone(), 25));
+            let state = all_online(5, seed, link.clone(), BLOCKS_PER_EPOCH + 1);
```

Comment:
It would be great to update `all_online` + the unclean shutdown test to have a few participants that do not have a share in the first epoch.